stdlib/public/core/Availability.swift

```diff
@@ -45,3 +45,39 @@ public func _stdlib_isOSVersionAtLeast(
   return false._value
 #endif
 }
+
+
+/// Returns the version number for the Swift Standard Library that the code
+/// calling this was originally compiled with.
+///
+/// The version number is an arbitrary `Double` value that is monotonically
+/// increasing between toolchain releases. It does not correlate with toolchain
+/// or OS version numbers, and it is not a semantic version number.
+@_alwaysEmitIntoClient @_transparent
```

Comment:
Why transparent? If we always build StdlibUnittest with a just-built stdlib, why AEIC?

Reply:
`@_transparent` is unnecessary, but removing `@_alwaysEmitIntoClient` would defeat the purpose of this symbol -- we want the actual number to be compiled into client binaries.

I added this to simplify detection of forward/backward deployment scenarios without baking the "current" version number into tests.